app/src/main/java/com/devmob/alaya/ui/screen/patientHome/PatientHomeScreenViewmodel.kt

```diff
@@ -0,0 +1,31 @@
+package com.devmob.alaya.ui.screen.patientHome
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import java.util.Calendar
+
+class PatientHomeScreenViewmodel():ViewModel() {
+    var namePatient by mutableStateOf("")
+    var greetingMessage by mutableStateOf("")
+
+    init {
+        fetchPatient()
+        updateGreetingMessage()
+    }
+
+    private fun fetchPatient() {
+        namePatient = "Flor"
+    }
+
+    private fun updateGreetingMessage() {
+        val calendar = Calendar.getInstance()
+        val hourOfDay = calendar.get(Calendar.HOUR_OF_DAY)
+        greetingMessage = when (hourOfDay) {
+            in 0..11 -> "Buenos días"
```

Comment:
A la 1 de la mañana diría "Buenos días"? Seria mejor lo siguiente?
`in 5..11 -> "Buenos días"`

Reply:
Tenes razon! lo cambio y lo subo nuevamente